Assets/Tests/InputSystem/CoreTests_Controls.cs

```diff
@@ -1119,6 +1119,34 @@ public void Controls_CanFindControlsUsingWildcards_InMiddleOfNames()
         }
     }
 
+    [Test]
+    [Category("Controls")]
+    public void Controls_MatchingPathToMask()
+    {
+        Assert.That(InputControlPath.BindingPathMatchesMask("<Keyboard>/e", "<Keyboard>/e"), Is.True);
+        Assert.That(InputControlPath.BindingPathMatchesMask("<Keyboard>/d", "<Keyboard>/e"), Is.False);
+    }
+
+    [Test]
+    [Category("Controls")]
+    public void Controls_MatchingPathToMaskWithWildcardAtBeginning()
+    {
+        Assert.That(InputControlPath.BindingPathMatchesMask("*/e", "<Keyboard>/e"), Is.True);
+        Assert.That(InputControlPath.BindingPathMatchesMask("*/d", "<Keyboard>/e"), Is.False);
+        Assert.That(InputControlPath.BindingPathMatchesMask("*/d", "<Keyboard>/d"), Is.True);
+        Assert.That(InputControlPath.BindingPathMatchesMask("*/leftButton", "<Mouse>/leftButton"), Is.True);
+    }
+
+    [Test]
+    [Category("Controls")]
+    public void Controls_MatchingPathToMaskWithWildcardAtEnd()
+    {
+        Assert.That(InputControlPath.BindingPathMatchesMask("<Keyboard>/*", "<Keyboard>/e"), Is.True);
```

Comment:
Could we have a test that checks the higher level MatchesMask function too ?
I.e. covers the flow where name, action or group is non null